obc.py

```diff
@@ -23,7 +23,12 @@ def start(self):
 
         # start apps
         imageService.start()
+<<<<<<< HEAD
+        videoDisplay.start()
```

Comment:
This line should be removed as well since there is no longer a `videoDisplay` import

Reply:
@logankuzyk Heads up, I think you still have to remove `videoDisplay.start()`